src/components/TransactionForm.tsx

```diff
@@ -61,7 +62,7 @@ const TransactionForm = ({ budget, setNotification }: Props) => {
     useState<UnpersistedAccount>()
   const [destinationAccountToCreate, setDestinationAccountToCreate] =
     useState<UnpersistedAccount>()
-  const [recentEnvelopes, setRecentEnvelopes] = useState([] as UUID[])
+  const [recentEnvelopes, setRecentEnvelopes] = useState([] as RecentEnvelope[])
```

Comment:
I'm not sure why we are using this syntax with `as` instead of
```suggestion
  const [recentEnvelopes, setRecentEnvelopes] = useState<RecentEnvelope[]>([])
```
🤔

Reply:
Because I didn't know better? 🤔